selftests/functional/basic.py

```diff
@@ -1093,6 +1101,17 @@ def test_non_absolute_path(self):
             f"Avocado did not return rc {expected_rc}:\n{result}",
         )
 
+    @skipUnlessPathExists("/bin/sh")
```

Comment:
I'm assuming `/bin/sh` here is supposed to match `EXEC_ENV_VARIABLE_TEST`, but `/bin/sh` and `/bin/bash` are not the same on every platform.